libnetwork/drivers/bridge/rootlesskit_client_linux.go

```diff
@@ -0,0 +1,131 @@
+// RootlessKit integration - if required by RootlessKit's port driver, let it know
+// about port mappings as they're added and removed.
+//
+// This is based on / copied from rootlesskit-docker-proxy, which was previously
+// installed as a proxy for docker-proxy:
+// https://github.com/rootless-containers/rootlesskit/blob/4fb2e2cb80bf13eb28b7f2a4317b63406b89ad32/cmd/rootlesskit-docker-proxy/main.go
+
+package bridge
```

Comment:
Probably this should be a separate package

Reply:
Done - I made it a package under `bridge`, not sure if that was the right choice.